middleware.js

```diff
@@ -49,6 +61,24 @@ module.exports.fetchRandomGameData = async (req, res, next) => {
   }
 };
 
+module.exports.fetchRandomGameDataArr = async (req, res, next) => {
+  const pageNumArr = generateUniqueRandomArr();
+  let gameDataArr = [];
+  for (let j = 0; j < 5; j++) {
```

Comment:
Honestly I sat for 5 minutes trying to understand what number 5 signifies here :D I would recommend to declare the number in a constant and come up with a clear name.

Reply:
this was here mostly for testing purposes, since i havent properly implemented the play functionality yet, decided just to pick up a few of the games and  get their data xd

Reply:
True, there are workarounds. You could try to use a modern `for of` loop instead - https://www.javascripttutorial.net/es6/javascript-for-of/